datafusion/physical-expr/src/planner.rs

```diff
@@ -106,6 +106,14 @@ pub fn create_physical_expr(
     input_dfschema: &DFSchema,
     execution_props: &ExecutionProps,
 ) -> Result<Arc<dyn PhysicalExpr>> {
+    use datafusion_physical_expr_common::physical_expr::create_physical_expr as create_physical_expr_common;
+
+    // Temporary solution, after all the logic is moved to common, we can remove this function
```

Comment:
Can you add a ticket reference in this comment too so anyone else who stumbles on this can review the backstory / reference?